server/routes/testutils/appSetup.ts

```diff
@@ -94,7 +107,7 @@ function appSetup({
 
   app.use((req, res, next) => {
     res.locals = {}
-    res.locals.user = req.user
+    res.locals.user = user
```

Comment:
Bug here that was fixed in the template app soon after we started (https://github.com/ministryofjustice/hmpps-template-typescript/pull/62)! The `user` defined above in this file for testing wasn't actually being used.

This change has led to lots of other test files in this PR having changes to 'user' from `undefined` to `user1`.

Reply:
Well spotted!